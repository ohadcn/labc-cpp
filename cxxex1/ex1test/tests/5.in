-298,799
39,527
681,352
427,-160
-239,-562
400,436
-823,17
-484,565
-907,65
606,363
-212,-579
-453,68
-679,-141
-935,-586
255,454
962,957
253,0
40,-510
-649,-533
330,-331
-538,-714
663,-361
303,-822
-797,397
243,367
-241,588
788,306
-344,666
-836,-722
-921,-24
733,-960
-510,-458
-402,530
32,-51
554,919
176,-985
-796,-162
-788,64
-427,973
-540,374
-104,220
-39,-317
-918,174
-652,805
453,985
-663,185
583,-173
284,-820
915,-685
-314,468
791,862
41,995
-744,-190
58,-170
-661,76
-797,236
853,164
919,492
-662,825
296,791
366,634
-468,-52
-983,816
-315,932
688,-71
-43,35
-653,998
29,-396
-636,644
-9,-297
-724,194
939,-872
915,415
621,-748
239,-527
600,-839
-894,689
-333,682
61,-91
613,-695
838,-431
340,-815
-877,-632
-654,-513
570,337
191,846
-913,129
532,-999
-457,152
812,339
625,411
58,289
-343,725
-30,718
-367,-861
-977,-972
-291,363
771,-168
289,-884
320,-585
