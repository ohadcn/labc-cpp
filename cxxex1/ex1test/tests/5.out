result
-983,816
-977,-972
-653,998
41,995
453,985
532,-999
733,-960
939,-872
962,957
